### ISSUE TITLE
After updating to v8 and changing timezone to pst Vancouver it has stopped updating my spreadsheet

### ISSUE THREAD

**Issue by Robro47** [FunOrange/circle-tracker#16]
When I updating circle tracker to v8 it gave me a message which asked if my timezone was Toronto, I clicked no and it gave me instructions on how to get the right timezone which i proceeded to follow. After about 20 minutes of playing I noticed that the spreadsheet was still at 0 hits on everything today. Not sure what to do here.
![image](https://user-images.githubusercontent.com/62081767/88495477-e09d9f80-cf6e-11ea-83a5-54bb567fe7af.png)
![image](https://user-images.githubusercontent.com/62081767/88495487-e5faea00-cf6e-11ea-9cf7-8b8ab62268fa.png)

**Comment by FunOrange** [comment]
Did you move around any of the columns in Raw Data, or rename the sheet? Also, when you go to Data > Named Ranges, can you make sure that all the variables point to the right columns?

**Comment by Robro47** [comment]
As far as I can tell I did not rename the sheet or move around any of the columns. I'm not sure what you mean by variables point to the right column but if u meant when I click on the variable it leads to the same name as the variable then yes it does (acc -> Acc).

**Comment by dcvasquez1** [comment]
I found that my spreadsheet was also not updating. Pretty sure it was an issue with the number of rows in "Raw Data". I added more rows (by clicking the add [___] more rows button at the bottom of the page) expecting the problem to be fixed but it still didn't work. I think it has to do with the row range of the variable "hits" within the raw data file but I don't know enough about Google APIs to fix it (my hamfisted attempt to just extend the range by putting in a bigger number didn't work)
